### PR TITLE
Support (C, 1, 1) for fill argument

### PR DESCRIPTION
**Pull request by Hakuyume**

### CODE REVIEW

File: chainercv/transforms/image/random_expand.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ def random_expand(img, max_ratio=4, fill=0, return_param=False):
             paper, this value is 4.
         fill (float, tuple or ~numpy.ndarray): The value of padded pixels.
```
Comment (yuyu2172) [inline chainercv/transforms/image/random_expand.py]: float or ~numpy.ndarray

File: chainercv/transforms/image/resize_contain.py [hunk 2]
```diff
@@ -19,6 +19,9 @@ def resize_contain(img, size, fill=0, return_param=False):
         size (tuple of two ints): A tuple of two elements:
             :obj:`height, width`. The size of the image after resizing.
         fill (float, tuple or ~numpy.ndarray): The value of padded pixels.
```
Comment (yuyu2172) [inline chainercv/transforms/image/resize_contain.py]: ditto

File: chainercv/transforms/image/random_expand.py (hunk 1 above)
Comment (Hakuyume) [inline chainercv/transforms/image/random_expand.py]: I think supporting tuple is convenient. Why do you want to drop it?

Comment (yuyu2172) [inline chainercv/transforms/image/random_expand.py]: Let me summarize.

`fill` can be one of `float, tuple or (C, 1, 1)`.
`mean` is `(C, 1, 1)`.

Comment (Hakuyume) [inline chainercv/transforms/image/random_expand.py]: > fill can be one of float, tuple or (C, 1, 1).
mean is (C, 1, 1).

Yes. And I want to pass a tuple value that is never broadcasted to an image.

**Comment by yuyu2172** [comment]
LGTM

**Pull request merged**